README.md

```diff
@@ -18,9 +18,10 @@ For information about how to get started with React Native SDK, please check thi
 
 ### Clone Quickstart app
 
-### Install Dependencies
+### Setup the project
```

Comment:
Please follow the README structure from https://github.com/hypertrack/quickstart-react-native or update it everywhere